plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/ArtifactHandler.kt

```diff
@@ -265,7 +309,46 @@ class ArtifactHandler(private val project: Project, private val clientAdaptor: G
             dialog.isModal = true
 
             if (dialog.showAndGet()) {
-                telemetry.viewArtifact(CodeTransformArtifactType.ClientInstructions, jobId, "Submit", source)
+                projectCoroutineScope(project).launch {
+                    telemetry.viewArtifact(CodeTransformArtifactType.ClientInstructions, jobId, "Submit", source)
+                    if (diffDescription == null) {
+                        val message = "I applied the changes to your project."
+                        val resultContent = CodeTransformChatMessageContent(
+                            type = CodeTransformChatMessageType.PendingAnswer,
+                            message = message,
+                        )
+                        codeTransformChatHelper?.updateLastPendingMessage(resultContent)
+                        codeTransformChatHelper?.addNewMessage(buildStartNewTransformFollowup())
+                    } else {
+                        if (getCurrentPatchIndex() < totalPatchFiles) {
+                            val message = "I applied the changes in diff patch ${getCurrentPatchIndex() + 1} of $totalPatchFiles. " +
+                                "${patchDescriptions[diffDescription.name]}"
+                            val notificationMessage = "Amazon Q applied the changes in diff patch ${getCurrentPatchIndex() + 1} of $totalPatchFiles " +
+                                "to your project."
+                            val notificationTitle = "Diff patch ${getCurrentPatchIndex() + 1} of $totalPatchFiles applied"
+                            setCurrentPatchIndex(getCurrentPatchIndex() + 1)
+                            notifyStickyInfo(notificationTitle, notificationMessage, project)
```

Comment:
this seems like it would be very annoying. why not have the variant that times out?